TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -1,50 +1,24 @@
 const Parser = require( './Parser' );
 const BenchmarkMetricRouter = require( './BenchmarkMetricRouter' );
 const BenchmarkMetric = require( './BenchmarkMetric' );
-
-const benchmarkDelimiterRegex = /\*\*\*\*\*\*\*\*\*\* START OF NEW TESTCI BENCHMARK JOB \*\*\*\*\*\*\*\*\*\*[\r\n]/;
-const benchmarkNameRegex = /Benchmark Name: (.*) Benchmark Variant: .*[\r\n]/;
-const benchmarkVariantRegex = /Benchmark Name: .* Benchmark Variant: (.*)[\r\n]/;
 const productResourceRegex = /Product Resource: (.*)[\r\n]/;
 
+const regexRunningTest = /Running test * \.\.\.[\r\n]/;
+let testIndex = 1; //interation 
 class BenchmarkParser extends Parser {
 
     // True iff all benchmarks and their corresponding iterations in the given run can be parsed
-    static canParse( buildName, output ) {
-        return benchmarkDelimiterRegex.test(output);
-    }
-
-    // Returns an iterator object which accesses each benchmark iteration from the given output
-    static getBenchmarkIterator(output) {
-        let index = 1;
-        let splitOutput = output.split(benchmarkDelimiterRegex);
-        let splitOutputLength = splitOutput.length;
-    
-        // No delimiter inside the output
-        if (!(Array.isArray(splitOutput)) || splitOutputLength === 1) {
-            return null;
+    static canParse( benchmarkName, benchmarkVariant ) {
+        if (benchmarkName && benchmarkVariant){
+            return true;
         }
```

Comment:
This would almost always be true since we'll have always have a `testName` from which you're getting `benchmarkName` and `benchmarkVariant` in `Test.js`.